include/dlaf/communication/kernels/broadcast.h

```diff
@@ -104,8 +102,11 @@ struct ScheduleRecvBcast {
     tile_cpu = std::move(pika::get<0>(pika::split_future(
         ScheduleRecvBcast<T>::call(ex, std::move(tile_cpu), root_rank, std::move(pcomm)))));
 
-    dataflow(getCopyExecutor<Device::GPU, Device::CPU>(), matrix::unwrapExtendTiles(copy_o), tile_cpu,
-             tile_gpu);
+    pika::execution::experimental::when_all(dlaf::internal::keepIfSharedFuture(std::move(tile_cpu)),
```

Comment:
This closes https://github.com/eth-cscs/DLA-Future/issues/448. This seems to not be an issue anymore.

Reply:
Actually, this does: https://github.com/eth-cscs/DLA-Future/pull/460/files#r812688112.

Reply:
And note, I changed this to sender adaptors in this PR to make sure that the old CUDA pools were not used in the benchmarks anymore (it turned out not to really make a difference, but it needed to be done anyway).